ecommerce/frontend/src/components/CategoryBar.js

```diff
@@ -0,0 +1,17 @@
+import { Menu } from 'antd'
+import { Link } from 'react-router-dom'
+
+export const CategoryBar = ({ categories = [] }) => {
```

Comment:
Although it has no problem, in the future, we may need to add sub sub categories, so a dropdown feature may be added to Category bar.

Reply:
I thinks it's a good consideration but it's not immediately needed as you have said.

Reply:
We will have to discuss this further since the requirements mention 17 categories. I don't remember seeing subcategories so probably some of them will become subcategories? I am not sure.

We can leave this for another PR.